capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
Capitalize the first letter of the given name, not upcase.

Reply:
@keneogu fixed
![e83](https://user-images.githubusercontent.com/40554384/180282047-83c6a90c-3e3a-4652-abae-a9d8f11621c3.JPG)